frontend/src/components/common/AddAdminModal.tsx

```diff
@@ -15,14 +17,34 @@ import {
   Text,
   useDisclosure,
 } from "@chakra-ui/react";
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+
+import AdminConfirmationMessage from "./Admin/AdminConfirmationMessage";
 import EmailInput from "./Form/EmailInput";
 import InputError from "./Form/InputError";
 import InputLabel from "./Form/InputLabel";
 import TextInput from "./Form/TextInput";
 import ModalFooterButtons from "./ModalFooterButtons";
 import { PlusOutlineIcon } from "./icons";
+import { ADMIN_PAGE } from "../../constants/Routes";
+import { UserRequest, UserResponse } from "../../types/UserTypes";
+import GET_USERS_BY_ROLE from "../../APIClients/queries/GetUsersByRole";
+
+const ADD_ADMIN = gql`
```

Comment:
given that this mutation is called `ADD_ADMIN`, should we set `role = "Admin"` as a default valut? Screenshot attached for code reference on what I mean!

<img width="943" alt="Screen Shot 2022-11-29 at 8 30 51 PM" src="https://user-images.githubusercontent.com/41273486/204685475-1aeb67d3-cb09-48bc-a354-50d705d2ea26.png">